spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/features/TaskConfiguration.java

```diff
@@ -126,7 +125,7 @@ public TaskRepositoryInitializer taskRepositoryInitializerForDefaultDB(DataSourc
 		@Bean
 		@ConditionalOnMissingBean
 		public TaskDefinitionRepository taskDefinitionRepository(DataSource dataSource, Server server) throws Exception {
```

Comment:
or if this is intentional, because this bean requires that the `Server` has been instantiated already, that intention would be clearer via `@DependsOn`... and that would make it less likely that someone deletes it at some point ;)